backend/models/helper/model.js

```diff
@@ -903,11 +903,11 @@ async function getMeshById(account, model, meshId) {
 		"_extRef":1
 	};
 
-	const mesh = await Scene.getObjectById(account, model, utils.stringToUUID(meshId), projection);
-	mesh.matrix = await Scene.getParentMatrix(account, model, mesh.parents[0], revisionIds);
+	const mesh = await getObjectById(account, model, utils.stringToUUID(meshId), projection);
```

Comment:
Can we rename the function getNodeById? we refer objects to different things really, In a 3D Scene we have nodes